waifus/madoka yabuuchi

```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "Madoka yabuuchi",
+	"image" : "https://cdn.myanimelist.net/images/characters/8/379321.jpg",
+	"japanese-name" : "藪内 円",
+	"year" : 2019,
+	"month" : 08,
```

Comment:
This is invalid JSON, the JSON parser doesn't understand what 08 means. Replace this with 8.